packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -3177,32 +3186,37 @@ export default class Meeting extends StatelessWebexPlugin {
 
   /**
    * Sets the local media stream on the class and emits an event to the developer
-   * @param {MediaStreamTrack} localShareTrack the local media stream
+   * @param {MediaStreamTrack} rawLocalShareTrack the local share media track
    * @returns {undefined}
    * @public
    * @memberof Meeting
    */
-  public setLocalShareTrack(localShareTrack: MediaStreamTrack) {
-    let settings = null;
+  public setLocalShareTrack(rawLocalShareTrack: MediaStreamTrack | null) {
+    if (rawLocalShareTrack) {
+      const settings = rawLocalShareTrack.getSettings();
+
+      const localDisplayTrack = new LocalDisplayTrack(
+        MediaUtil.createMediaStream([rawLocalShareTrack])
```

Comment:
same here

Reply:
https://github.com/webex/webrtc-core/blob/beta/packages/webrtc-core/src/media/track.ts#L76

Reply:
I'm using webrtc-core from master for now and the constructor there takes a stream: https://github.com/webex/webrtc-core/blob/main/src/media/local-track.ts#L84